### PR TITLE
Release update

### PR DESCRIPTION
**Pull request by Hebaallahibrahim**
This pull request for updating the inference layer for the new release.

### CODE REVIEW

File: sansa-inference-spark/pom.xml [hunk 1]
```diff
@@ -230,6 +231,7 @@
         <dependency>
             <groupId>org.apache.commons</groupId>
             <artifactId>commons-compress</artifactId>
+            <version>1.19</version>
```
Comment (LorenzBuehmann) [inline sansa-inference-spark/pom.xml]: why here and not in SANSA-Parent?

Comment (Hebaallahibrahim) [reply sansa-inference-spark/pom.xml]: ok, Could you please add this version to SANSA-Parent?

Comment (GezimSejdiu) [reply sansa-inference-spark/pom.xml]: It is already in SANSA-Parent: 

https://github.com/SANSA-Stack/SANSA-Parent/blob/bccf80a80e5a06095de2ee62a4f3cd496b222870/pom.xml#L124,L128

Can you rebuild to see if the changes are reflected?

Comment (patrickwestphal) [reply sansa-inference-spark/pom.xml]: Yep, seems to work.

**Pull request merged**